components/stripe/actions/create-invoice/create-invoice.mjs

```diff
@@ -1,98 +1,133 @@
-import pick from "lodash.pick";
 import app from "../../stripe.app.mjs";
 
 export default {
   key: "stripe-create-invoice",
   name: "Create Invoice",
   type: "action",
-  version: "0.1.1",
-  description: "Create an invoice. [See the docs](https://stripe.com/docs/api/invoices/create) " +
-    "for more information",
+  version: "0.1.2",
+  description: "Create an invoice. [See the docs](https://stripe.com/docs/api/invoices/create) for more information",
```

Comment:
Can we do a find/replace all from "see the docs" to "see the documentation"? We're usually avoiding the abbreviations in descriptions